features/transferable-streams.yml

```
@@ -0,0 +1,8 @@
+name: Transferable streams
+description: Streams are transferable objects, which can be moved between contexts such as workers.
```

"Between" implies two things and I think making it explicit is nice. Does this wording work for you?

```suggestion
description: Streams are transferable objects, which can be moved between contexts such as windows and workers.
```